src/Azs.AzureBridge.Admin/test/Get-AzsAzureBridgeDownloadedProduct.Tests.ps1

```diff
@@ -1,3 +1,19 @@
+$envFile = 'env.json'
```

Comment:
can this be abstracted to a common file instead of repeating it in all the tests?

Reply:
I tried moving it to common file, but both modules started failing with an unrelated error, so I moved it back to each file.

Reply:
From Xiaogang:

> If you need to add a default parameter, you should do it in the setEnv function in utils.ps1.
>  
> You may refer to https://github.com/Azure/azure-powershell/blob/generation/src/Databricks/test/utils.ps1 as an example.

If we use this method, then we won't need to add the above lines of code for every test.

Reply:
Also, this is only necessary if you are adding a default parameter other than subscription id and tenant id. For example, location. Otherwise you can revert back to importing localEnv.ps1.